src/Sulu/Bundle/ContactBundle/Controller/AbstractMediaController.php

```diff
@@ -160,6 +160,7 @@ protected function getMultipleView($entityName, $routeName, AbstractContactManag
 
                 $listBuilder = $factory->create($entityName);
                 $fieldDescriptors = $this->getFieldDescriptors($entityName);
+                $listBuilder->setIdField($fieldDescriptors['id']);
```

Comment:
Ok, I see why this is necessary... We can't join the other way round... Don't really like it, but I also don't see another fast solution...